remember.py

```diff
@@ -0,0 +1,30 @@
+"""
+Requieremenrs : PIL Package
+
+This program gets path to a file, then is takes the row number of each black pixel,
+then uses the chr() function to convert it to a letter.
+@param : path - path to a png file
+@return : string
+"""
+
+from PIL import Image
+
+
+def decode_png(path):
```

Comment:
Write documentation in the right format - 
python
def count_hi_v4(word: str) -> int:
    """
    The function counts how many times the string "hi" appears in a given string.
    @param word: The given string.
    @return: The amount of times the word "hi" appeared.
    """
    return word.count("hi")